CHANGELOG.md

```diff
@@ -9,17 +9,26 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Adoption by Sous-Chefs, adds CircleCI configuration for supported Os
+- CentOS 8 to test-kitchen platforms and include CentOS to workflow matrix
+- Debian 10 & Ubuntu 20.04 to test-kitchen platforms and workflow matrix
+- Amazon Linux 2 test-kitchen platforms and workflow matrix
+- 'socket' and 'socket_mode' property to the resource
+- ChefSpec tests for test::instance recipe
 
 ### Changed
 
-- Systemd service will be used when systemd is available if not then sysv init is used
+- Switched to using systemd_unit resource
 - migrated to github actions
+- Update and improved ChefSpec tests
+- Move library helper functions into a Memcached::Helpers namespace
 
 ### Removed
 
-- Support / testing for Ubuntu 14.04
+- Support / testing for Ubuntu 14.04, CentOS 6 & Debian 8
```

Comment:
Debian 8 should probably just be in the testing section. It should work with the systemd stuff even if it's untested